prompting/validators/reward/open_assistant.py

```diff
@@ -40,20 +40,22 @@ def __init__(self, device: str):
             OpenAssistantRewardModel.reward_model_name
         ).to(self.device)
 
-    def reward_single(self, prompt: str, completion: str, name: str) -> float:
+    def reward_single(self, prompt: str, completion: str, name: str) -> BaseRewardEvent:
+        reward_event = BaseRewardEvent()
+
         with torch.no_grad():
             inputs = self.tokenizer(prompt, completion, return_tensors="pt").to(
                 self.device
             )
-            return float(self.model(**inputs).logits[0].cpu().detach())
+            reward_event.reward = float(self.model(**inputs).logits[0].cpu().detach())
```

Comment:
shouldn't this be `reward_event.score` since its float? where is `reward_event.reward:Bool`?

Reply:
rlhf is a reward model (not filter model), so there is no bool reward, and the score = reward